runners/spark/src/main/java/org/apache/beam/runners/spark/translation/GroupCombineFunctions.java

```diff
@@ -131,7 +128,13 @@
               return CoderHelpers.toByteArray(merged, iterAccumCoder);
             });
 
-    return Optional.of(CoderHelpers.fromByteArray(accumulatedBytes, iterAccumCoder));
+    final Iterable<WindowedValue<AccumT>> result =
+        CoderHelpers.fromByteArray(accumulatedBytes, iterAccumCoder);
+    if (Iterables.isEmpty(result)) {
```

Comment:
can you inline this one at the rebase moment please too `return Iterables.isEmpty(result) ? Optional.absent() : Optional.of(result);`.

Reply:
done